scripts/run-pelorus-e2e-tests

```diff
@@ -100,14 +100,24 @@ function print_help() {
     printf "\tStartup:\n"
     printf "\t  -h\tprint this help\n"
     printf "\n\tOptions:\n"
-    printf "\t  -b\tbranch of the mig-demo-apps\n"
-    printf "\t  -f\tvalues filename passed to the Pelorus deployment from the todolist-mongo-go project\n"
-    printf "\t  -o\tgithub organization of the mig-demo-apps\n"
-    printf "\t  -d\tpath to virtualenv DIR\n"
-    printf "\t  -s\tpath to DIR with secrets files (used in prow CI)\n"
-    printf "\t  -e\tenable exporter <comma_list>. e.g. failure\n"
-    printf "\t  -t\tenable Thanos with s3 bucket\n"
+    printf "\t  -o\tgithub organization of the mig-demo-apps. By default, konveyor\n"
+    printf "\t  -b\tbranch of the mig-demo-apps. By default, master\n"
+    printf "\t  -f\tvalues filename of the mig-demo-apps. By default, values.yaml\n"
+    printf "\t  -d\tpath to Python virtual environment DIR. By default, the project's one\n"
+    printf "\t  -s\t(USED ONLY IN PROW CI) path to DIR with secrets files. For local runs, run 'export ENVIRONMENT_VARIABLE_NAME=VALUE'\n"
+    printf "\t  -t\tenable Thanos (long term persistent storage) tests with s3 bucket\n"
     printf "\t  -c\tpath to DIR with secrets files for s3 bucket\n"
+    printf "\t  -e\tenable exporter tests (comma separated). Examples:\n"
+    printf "\t\t  -e failure\n"
+    printf "\t\t  -e gitlab_committime,jira_committime\n"
+    printf "\t\tavailable options:\n"
+    printf "\t\t  failure - GitHub issue tracker - REQUIRES 'TOKEN=YOUR_GITHUB_TOKEN' ENVIRONMENT VARIABLE\n"
```

Comment:
That is misleading.

Each of the options does not *require* environment variable. It is an option to have environment variable *or* a filename with the credentials in a given directory. See function: `function create_k8s_secret`